src/scripts/main.js

```diff
@@ -1,7 +1,35 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector("body")
+  const message = document.createElement("div")
+  message.classList.add('notification')
```

Comment:
You can add multiple classes to an element at once by passing them as arguments to `classList.add`. For example, `message.classList.add('notification', type);`. This makes the code more concise.